src/EasyAbp.BookingService.Application.Contracts/EasyAbp/BookingService/AssetPeriodSchemes/Dtos/CreateUpdateAssetPeriodSchemeDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using Volo.Abp.ObjectExtending;
+
+namespace EasyAbp.BookingService.AssetPeriodSchemes.Dtos;
+
+[Serializable]
+public class CreateUpdateAssetPeriodSchemeDto : ExtensibleObject
```

Comment:
Missing `AssetId` and `Date`.

Reply:
`AssetId` and `Date` is "Key of entity"?

Reply:
They are used to build the key of an entity.